sharded_queue/api.lua

```diff
@@ -468,13 +468,10 @@ function sharded_queue.create_tube(tube_name, options)
 end
 
 local function init(opts)
-    if opts.is_master then
-        rawset(_G, 'queue', sharded_queue)
-    end
+    rawset(_G, 'queue', sharded_queue)
 end
 
 local function apply_config(cfg, opts)
-    if opts.is_master then
```

Comment:
Since you remove `if` you need to reduce indent inside this section.

Reply:
done